internal/controllers/core/uibutton/reconciler.go

```diff
@@ -44,6 +47,9 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		return ctrl.Result{}, nil
```

Comment:
I think you're missing a dispatch for the `UIButtonDeleteAction`  in this block

Reply:
😬